### PR TITLE
Fr 1762 notify ctsc

### PR DESCRIPTION
**Pull request by HarryH96**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FR-1762

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -61,6 +61,8 @@ finrem.notification.assignToJudge=/assign-to-judge
 finrem.notification.consentOrderMade=/consent-order-made
 finrem.notification.consentOrderNotApproved=/consent-order-not-approved
 finrem.notification.consentOrderAvailable=/consent-order-available
+finrem.notification.consentOrderAvailableCtsc=/consent-order-available-ctsc
+finrem.notification.ctscEmail=${CTSC_EMAIL:fr_applicant_sol@sharklasers.com}
```
Comment (ocarleton) [inline src/main/resources/application.properties:65]: do we need a story or task for release for setting this to the courts address?

Comment (ocarleton) [reply src/main/resources/application.properties:65]: nevermind, i see it in cnp flux config for prod 👍

Comment (shaunganley) [reply src/main/resources/application.properties:65]: (see Flux PR below)

File: charts/finrem-cos/values.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ java:
     IDAM_API_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     IDAM_S2S_URL : "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     DOCUMENT_GENERATOR_SERVICE_API_BASEURL : "http://finrem-dgcs-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
-    FINREM_NOTIFICATION_SERVICE_BASE_URL : "http://finrem-ns-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    FINREM_NOTIFICATION_SERVICE_BASE_URL : "http://finrem-ns-pr-164.service.core-compute-preview.internal"
```
Comment (HarryH96) [inline charts/finrem-cos/values.yaml:11]: Reminder to revert before merge :)

**Pull request merged**